run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Public Functions
+
+// RunContainer runs a container with given ID
+func RunContainer(containerID string, containerPath string) error {
+	conn, err := dbus.NewSystemConnection()
+	if err != nil {
+		return err
+	}
+
+	return runContainerInternal(containerID, containerPath, conn)
+}
+
+// Internal Functions
+
+// Produce a per-container unique name used to identify its transient systemd service
+func getUnitName(containerID string) string {
+	return fmt.Sprintf("kpod-container-%s.service", containerID)
```

Comment:
My understanding was kpod was about using images and pods, so I definitely think you want to be able to schedule a pod within a systemd unit file. You might be running an httpd container with a monitoring agent in the same pod.  Why not schedule them together.  

The goals of these tools is to "debug" what kubernetes is doing, so I believe we should work with pods, even if 99.99% of the time they are just single containers running within the pod.